<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_auto</class>
 <widget class="QDialog" name="form_auto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QSlider" name="horizontalSlider_auto_velocity_procent">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_auto_velocity_procent">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>auto speed -100 - 0 - 100%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_tooldir_fillet"/>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>187</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QToolButton" name="toolButton_reload">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string> file reload</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/pictures/btn_25_25_file_refresh.png</normaloff>:/pictures/btn_25_25_file_refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="spinBox_run_from_line"/>
     </item>
     <item row="3" column="0">
      <widget class="QToolButton" name="toolButton_run">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string> run</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/pictures/btn_25_25.png</normaloff>:/pictures/btn_25_25.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QToolButton" name="toolButton_pause">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string> pause</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/pictures/btn_25_25.png</normaloff>
         <normalon>:/pictures/btn_25_25_in_orange.png</normalon>:/pictures/btn_25_25.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="toolButton_open">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string> file open</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/pictures/btn_25_25_file_open.png</normaloff>:/pictures/btn_25_25_file_open.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QToolButton" name="toolButton_stop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string> stop</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/pictures/btn_25_25.png</normaloff>
         <normalon>:/pictures/btn_25_25_in_purple.png</normalon>:/pictures/btn_25_25.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_auto">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>- AUTO -</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>From line</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>tooldir blend value mm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
